exporter/exporterhelper/metadata.yaml

```
@@ -107,17 +107,26 @@ telemetry:
       enabled: true
       stability:
         level: alpha
-      description: Current size of the retry queue (in batches)
+      description: Current size of the retry queue (in batches).
       unit: "{batches}"
       gauge:
         value_type: int
         async: true
 
+    exporter_queue_batch_size:
+      enabled: true
+      stability:
+        level: alpha
+      description: Size of the batches added to the retry queue (in bytes).
+      unit: "{By}"
```

```suggestion
      unit: "By"
```

Bytes is present in UCUM, so there's no need for brackets :) 
[We should use brackets only when the unit chose is fundamentally not a unit](https://opentelemetry.io/docs/specs/semconv/general/metrics/#units:~:text=Units%20should%20follow%20the%20Unified%20Code%20for%20Units%20of%20Measure.), e.g. {record} or {batch}